<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaEnsambladora</class>
 <widget class="QMainWindow" name="ventanaEnsambladora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ensambladora</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="espaciovacio">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="masareceta">
           <property name="text">
            <string>Masa(g)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="masachocolate">
           <property name="text">
            <string>Chocolate(g)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="masatiempo">
           <property name="text">
            <string>Tiempo(s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cantidad">
           <property name="text">
            <string>Cantidad/tiempo(s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalgalletas">
           <property name="text">
            <string>Total de Galletas</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Receta</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editorMasa">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editorChocolate">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editorTiempo">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editorCantidad">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalGalletas1">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Encendido</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Apagado</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="configurar">
       <property name="text">
        <string>Configurar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
